src/run/troubleshoot/analysis/DockerMemoryAnalysis.ts

```diff
@@ -0,0 +1,92 @@
+import { collect, deduplicate } from "../../../helpers/array-reducers.js";
+import {
+    fetchDockerSettings,
+    DockerSettings
+} from "../../../helpers/docker-settings-store.js";
+import Service from "../../../model/Service.js";
+import { Inventory } from "../../../state/inventory.js";
+import { StateManager } from "../../../state/state-manager.js";
+import AnalysisOutcome from "./AnalysisOutcome.js";
+import { AnalysisIssue, TroubleshootAnalysisTaskContext } from "./AnalysisTask.js";
+import os from "os";
+
+const ANALYSIS_HEADLINE = "Check docker's memory allocation size";
+
+const DOCKER_MEMORY_SUGGESTIONS = [
+    "Got to settings on Docker Desktop application",
+    "Select the Resource tab - Advanced",
+    "Add to Memory Limit using the slider"
```

Comment:
Add suggestions for where >10 services enabled - particularly where they have set 50% available memory and still <12gb. Maybe something like reduce number of services running.